CHANGELOG.md

```diff
@@ -80,6 +90,8 @@
  * **Donate button** - If you would like to show your appreciation with a monetary gift, please click the Donate button
  * **Updates** - Get updated on the latest SikhiToTheMax news
 
+<details><summary>See more...</summary>
```

Comment:
was this the intended result?
![image](https://user-images.githubusercontent.com/2982784/53059988-b7b9d680-346d-11e9-9ce8-d41baeb0ce62.png)

Reply:
This was a test to see how the formatting would look in the build. Looked fine on GitHub. 

I'll go back and remove that. Any suggestions to achieve something similar?

Reply:
No need to remove. Dependency needed updating. https://github.com/KhalisFoundation/sttm-desktop/pull/504